R/Spectra.R

```diff
@@ -83,6 +83,31 @@ NULL
 #' - parameter `BPPARAM`: setup for the parallel processing. See [bpparam()] for
 #'   details.
 #'
+#' Data from a `Spectra` object can be **exported** to a file with the `export`
+#' function. The file format for the exported data can be specified with the
+#' parameter `format`; currently only `format = "mgf"` is supported which
+#' exports the data in Mascot Generic Format (MGF). By default all non-empty
+#' spectra variables in `Spectra` are exported.
+#'
+#' The definition of the function is
+#' `export(object, file = tempfile(), format = "mgf",
+#'     variableMapping = spectraVariableMapping(format), ...)` and ist
```

Comment:
and *its*

Reply:
done in https://github.com/rformassspectrometry/Spectra/pull/125/commits/4be7b958b75c680ae660575ac1a53c81b7d7dfa7